src/app/hooks/useGlean.ts

```diff
@@ -28,6 +28,7 @@ export const useGlean = () => {
       // This will submit an events ping every time an event is recorded.
       maxEvents: 1,
       channel: PUBLIC_APP_ENV,
+      enableAutoPageLoadEvents: true,
```

Comment:
If page load events are enabled automatically, do we still need the `<PageLoadEvent>` tag in https://github.com/mozilla/blurts-server/blob/c07fd0a824b54abe93d89f73026a78730e3ef266/src/app/(proper_react)/layout.tsx#L32-L35

```sh
git grep -in "pAgeLoAd" | cat

src/app/(proper_react)/layout.tsx:14:import { PageLoadEvent } from "../components/client/PageLoadEvent";
src/app/(proper_react)/layout.tsx:32:          <PageLoadEvent
src/app/components/client/PageLoadEvent.tsx:20:export const PageLoadEvent = (props: Props) => {
src/app/functions/server/getUserId.tsx:27:      // This is set client-side in <PageLoadEvent>.
src/app/hooks/useGlean.ts:31:      enableAutoPageLoadEvents: true,
```

Reply:
Yes, we're actively using our custom pageload event for our analytics. We're helping to test the new built-in one, and will need to transition on the analysis side if we decide to move. Having both to compare in the meantime will be helpful.